runtime/src/enclave_rpc/client.rs

```diff
@@ -408,11 +471,15 @@ impl RpcClient {
         // Create the command channel.
         let (tx, rx) = mpsc::channel(CMDQ_BACKLOG);
 
+        // Ensure every client generates unique request IDs.
+        let last_request_id = Self::generate_random_request_id();
```

Comment:
Hm, this could in theory overlap? We could use a random ID for each request or a global atomic for these, but then the individual methods (`connect`, `call*`, etc.) would need to return the request ID instead of updating/reading from `self.last_request_id`. Or maybe the odds are small?

Reply:
> Hm, this could in theory overlap? 

In theory could overlap, but probability should be negligible for small number of clients. 

> We could use a random ID for each request

This should be the same. What you loose here is that requests from the same client don't have consecutive numbers.

>  global atomic

Not the best, as runtime restart starts with the same ids.

>  individual methods (`connect`, `call*`, etc.) would need to return the request ID instead of updating/reading from `self.last_request_id`

Don't want to return request from `transport`, as it complicates things and probably doesn't even belong there.

Change to `request_id = client_id << 32 + counter % (1 << 32)`, i.e. I use first 32 bits for client id (a random value on startup which is increased for every new client) and the last 32 bits for counter/call id (cycling counter when reaching max).